### PR TITLE
MON-3848 #comment parsing files via eslint

### PR DESCRIPTION
**Pull request by EdytaSki**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -42,19 +42,23 @@
...
+    "react-sortable-hoc": "^1.9.1",
     "redux": "^4.0.1",
     "redux-devtools-extension": "^2.13.8",
     "redux-form": "^8.1.0",
     "redux-saga": "^1.0.2",
     "redux-thunk": "^2.3.0",
     "replace-in-files": "^1.1.4",
-    "sass-loader": "^7.1.0"
+    "sass-loader": "^7.1.0",
+    "typescript": "^3.5.2"
```
[the remark this answers is not in the document]
Comment (EdytaSki) [reply package.json]: of course

File: src/FormTemplate/index.js [hunk 2]
```diff
@@ -0,0 +1,182 @@
...
+        autoComplete="on"
+        data={data}
+        modifier
+        {...rest}
+      >
+        <FormControl variant="outlined" className={classes.formControl}>
+          {data.map(form => {
+            switch (form.input_type) {
+              case 'dropdown':
+                return (
```
Comment (bdauria) [inline src/FormTemplate/index.js]: Do you think that would make sense to create sub-components for each case?

File: src/FormTemplate/index.js [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+                  <div key={form.id}>
+                    <InputLabel
+                      className={classes.selectLabel}
+                      ref={ref => {
+                        this.InputLabelRef = ref;
```
Comment (bdauria) [inline src/FormTemplate/index.js]: Why do we need the ref here?

File: src/SortableList/index.js [hunk 4]
```diff
@@ -0,0 +1,316 @@
...
+          forId: 'id2',
+          required: true
+        }
+      ]
+    };
+  }
+
+  handleChange = name => event => {
+    let source = event.target.value;
+    console.log(source);
```
Comment (bdauria) [inline src/SortableList/index.js]: no console.log :)

File: src/SortableList/index.js [hunk 5]
```diff
@@ -0,0 +1,316 @@
...
+          value: 'Regex pattern',
+          forId: 'id2',
+          required: true
+        }
+      ]
+    };
+  }
+
+  handleChange = name => event => {
+    let source = event.target.value;
```
Comment (bdauria) [inline src/SortableList/index.js]: it's a const

File: src/TablePagination/TableBody.js [hunk 6]
```diff
@@ -0,0 +1,162 @@
...
+  };
+
+  render() {
+    const { classes } = this.props;
+
+    return (
+      <React.Fragment>
+        <TableBody>
+          {jobs.map(job => {
+            return (
```
Comment (bdauria) [inline src/TablePagination/TableBody.js]: We can probably extract this into a JobRow Component

File: src/SortableList/index.js (hunk 4 above)
Comment (EdytaSki) [reply src/SortableList/index.js]: sorry ;)

File: src/TablePagination/TableBody.js (hunk 6 above)
Comment (EdytaSki) [reply src/TablePagination/TableBody.js]: not bad as idea ...

File: src/SortableList/index.js (hunk 5 above)
Comment (EdytaSki) [reply src/SortableList/index.js]: ok

File: src/FormTemplate/index.js (hunk 3 above)
Comment (EdytaSki) [reply src/FormTemplate/index.js]: it was implemented with MUI template but  true - it's working well without.

Comment (bdauria) [reply src/FormTemplate/index.js]: ok, so I guess we can get rid of it :)

File: src/TablePagination/TableBody.js (hunk 6 above)
Comment (EdytaSki) [reply src/TablePagination/TableBody.js]: As discussed I will do  this refacto a bit later once other priorities finished.

File: src/FormTemplate/index.js (hunk 2 above)
Comment (EdytaSki) [reply src/FormTemplate/index.js]: As discussed I will do  the refacto for select part a bit later once other priorities finished.

File: package.json [hunk 7]
```diff
@@ -42,12 +42,15 @@
     "loaders.css": "^0.1.2",
     "prop-types": "^15.6.2",
     "react": "^16.4.2",
+    "react-docgen-typescript-loader": "^3.1.0",
```
Comment (bdauria) [inline package.json]: Is that dependency needed?

**Comment by bdauria** [comment]
@EdytaSki What's the status here?

**Comment by bdauria** [comment]
@EdytaSki up

**Pull request closed without merging**